presentation/PresentationConnection_basic-manual.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<!--
```

Comment:
Please remove all license information from test files before submitting to W3C.

All W3C tests are licensed under http://www.w3.org/Consortium/Legal/2008/04-testsuite-copyright.html, see https://github.com/w3c/web-platform-tests/blob/master/LICENSE